runelite-client/src/main/java/net/runelite/client/util/Text.java

```diff
@@ -206,4 +210,49 @@ public static boolean matchesSearchTerms(String[] searchTerms, final Collection<
 		}
 		return true;
 	}
+
+	/**
+	 * Returns a formatted time string
+	 *
+	 * @param remainingSeconds
+	 * @param useRelativeTime
+	 * @return a relative time string (1d 2h 3m) if useRelativeTime is true, date time string (Sunday at 3:05) (at 17:30) if useRelativeTime is false
+	 */
+
+	public static String getFormattedEstimate(long remainingSeconds, boolean useRelativeTime)
```

Comment:
why did you move this?

Reply:
I couldn't think of a better way to be able to access this for the infobox.

Reply:
just make it public where it used to be?